tmva/tmva/Module.mk

```diff
@@ -57,6 +62,7 @@ TMVAH1       := $(patsubst %,$(MODDIRI)/TMVA/%,$(TMVAH1))
 TMVAH2       := $(patsubst %,$(MODDIRI)/TMVA/%,$(TMVAH2))
 TMVAH3       := $(patsubst %,$(MODDIRI)/TMVA/%,$(TMVAH3))
 TMVAH4       := $(patsubst %,$(MODDIRI)/TMVA/%,$(TMVAH4))
+TMVAH5       := $(patsubst %,$(MODDIRI)/TMVA/%,$(TMVAH5))
```

Comment:
These don't seem to be used... Did you actually need to add `TMVAH5` (which would contradict my assessment that they are not used!)

Reply:
Yes, but I had several problems to fix at the same time, I will try to trim it down to only the really needed things. However, users may need the headers even though they are not needed to compile ROOT, no?

Reply:
But the users are not using a Makefile variable called `TMVAH5`. I don't know of any user.

Reply:
Ok, I will review those changes again.